tests/test_circuit.py

```diff
@@ -1149,26 +1152,34 @@ def test_qiskit2tc():
 @pytest.mark.parametrize("backend", [lf("tfb"), lf("jaxb")])
 def test_qiskit2tc_parameterized(backend):
     try:
-        from qiskit.circuit import QuantumCircuit, Parameter
-        from qiskit.quantum_info import Operator
+        from qiskit.circuit import Parameter, ParameterVector, QuantumCircuit
         from qiskit.circuit.library import TwoLocal
+        from qiskit.quantum_info import Operator
         from qiskit_nature.second_q.circuit.library import UCCSD
-        from qiskit_nature.second_q.mappers import ParityMapper, QubitConverter
+        from qiskit_nature.second_q.mappers import ParityMapper
     except ImportError:
         pytest.skip("qiskit or qiskit-nature is not installed")
     from tensorcircuit.translation import perm_matrix
 
     mapper = ParityMapper()
-    converter = QubitConverter(mapper=mapper, two_qubit_reduction=True)
-    ansatz1 = UCCSD(2, [1, 1], converter)
+    ansatz1 = UCCSD(2, [1, 1], mapper)
     ansatz2 = TwoLocal(2, rotation_blocks="ry", entanglement_blocks="cz")
     ansatz3 = QuantumCircuit(1)
     ansatz3_param = Parameter("θ")
     ansatz3.rx(ansatz3_param, 0)
-    ansatz_list = [ansatz1, ansatz2, ansatz3]
+    ansatz4 = QuantumCircuit(1)
+    ansatz4_param = ParameterVector("φ", 3)
+    ansatz4.rx(2.0 * ansatz4_param[0] + 5.0, 0)
+    ansatz4.ry(ansatz4_param[0] * ansatz4_param[1] + ansatz4_param[2], 0)
+    ansatz4.rz(
+        np.exp(np.sin(ansatz4_param[0]))
+        + np.abs(ansatz4_param[1]) / np.arctan(ansatz4_param[2]),
```

Comment:
`ParameterVector` doesn't support  `np.abs` for qiskit with early versions, though not determine the exact breaking version (at least 0.23 fails). 
For now, I think it is ok to have `abs` in the test, just comment for a record.